src/patterns/task-list-pages/index.md

```diff
@@ -62,16 +60,16 @@ You might find a better word to describe what a ‘task’ is for your users. If
 
 ![A screenshot showing a task list summary that says 'Application incomplete. You've completed 3 of 8 sections'](task-list-summary.png)
 
-Make it clear to users which tasks they’ve completed and which still need their attention, by labelling them using the [Tag component](/components/tag/).
+Make it clear to users which tasks they’ve completed and which still need their attention, by labelling them using statuses.
 
-![A screenshot showing a completed tag next to a list of tasks](task-list-statuses.png)
+{{ example({group: "patterns", item: "task-list-pages", example: "task-list-statuses", html: true, nunjucks: true, open: false}) }}
```

Comment:
This example isn’t consistent with the guidance in the task list component guidance (or the pattern guidance) because it shows `Not started`, `In progress` and `Cannot start yet` in black, which is reserved for the `Completed` status.

Reply:
Fixed in 9aac0983dd792a0395fe9cd4b86330666ee1f964 (forgot to update this example when we changed how the css classes worked, opps).